.github/workflows/docker_publish.yaml

```diff
@@ -6,14 +6,8 @@ on:
       - main
     paths-ignore:
       - website/**
-    tags:
-      - 'v*'
+  workflow_call:
```

Comment:
Instead of relying on tags, publish docker from the release workflow: https://github.com/Unleash/unleash/pull/7089/files#diff-c15c3b26808da8f11c4b99dba3911b9ded250104a2c8991073e1993e90444612R111